Gopkg.toml

```diff
@@ -53,9 +53,51 @@ required = [
 # See here:
 #  - https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/116#discussion_r262599059
 #  - https://github.com/kubernetes-sigs/controller-runtime/pull/349
+
+[[constraint]]
+  name="sigs.k8s.io/controller-runtime"
+  version="v0.1.10"
+
 [[constraint]]
+  name="sigs.k8s.io/controller-tools"
+  version="v0.1.1"
+
+# For dependency below: Refer to issue https://github.com/golang/dep/issues/1799
+[[override]]
+  name = "k8s.io/kubernetes"
+  version = "=v1.13.3"
```

Comment:
We should be using `1.13.4` if we were to version pin at all.

Reply:
Yep! removed all overrides